core/trino-main/src/test/java/io/trino/block/BlockAssertions.java

```diff
@@ -21,6 +21,8 @@
 import io.trino.spi.block.RunLengthEncodedBlock;
 import io.trino.spi.type.ArrayType;
 import io.trino.spi.type.DecimalType;
+import io.trino.spi.type.MapType;
```

Comment:
Describe in the commit message what actually has been changed

Reply:
added more detailed description